frontend/src/v5/ui/routes/viewer/viewer.tsx

```diff
@@ -91,20 +92,11 @@ export const Viewer = () => {
 		return <InvalidFederationOverlay containers={federationsContainers} />;
 	}
 
-	const v4Match = {
-		params: {
-			model: containerOrFederation,
-			project,
-			teamspace,
-			revision,
-		},
-	};
-
 	return (
-		<>
+		<CalibrationContextComponent>
 			<OpenTicketFromUrl />
 			<CheckLatestRevisionReadiness />
-			<ViewerGui match={v4Match} key={containerOrFederation} />
-		</>
+			<ViewerCanvases />
```

Comment:
Why did you change  where viewercancases is? is there's a good reason talk to Dan about it